public/app/system.conf.js

```diff
@@ -1,6 +1,15 @@
+function getBaseURL() {
+  "use strict";
+  /* in the case of system.build, window is undefined
+     We should still pull the plugins from the CDN*/
+  if(typeof(window) === "undefined") {
+    return "public";
+  }
+  return window.grafanaBootData.settings.appStaticUrl || "public";
+}
 System.config({
   defaultJSExtenions: true,
-  baseURL: 'public',
+  baseURL: getBaseURL(),
```

Comment:
Dont we want to load those things from the static server? In this case, we need to have system.conf point to the static server.

So for the build, we need it to point to local filesystem 'public', but for the production, we need it to point to the static server.